.codegen.json

```diff
@@ -0,0 +1,15 @@
+{
+    "formatter": "go run golang.org/x/tools/cmd/goimports@latest -w $FILENAMES && gofmt -w $FILENAMES",
+    "version": {
+        "common/version.go": "version = \"$VERSION\""
+    },
+    "toolchain": {
+        "required": [
+            "go"
+        ],
+        "post_generate": [
+            "go run honnef.co/go/tools/cmd/staticcheck@v0.4.0 ./...",
+            "go run gotest.tools/gotestsum@latest --format pkgname-and-test-fails --no-summary=skipped --raw-command go test -v -json -short -coverprofile=coverage.txt ./..."
```

Comment:
Wouldn't it be easier to run the corresponding `make` command instead?

Reply:
Yup, testing that out now